lib/test/Test/Pos/Launcher/Json.hs

```diff
@@ -0,0 +1,158 @@
+                          (TxSizeLinear
+                              (Coeff 155381.000000000) (Coeff 43.946000000))
+                , bvdUnlockStakeEpoch = EpochIndex 1844
+                }
+            , gsProtocolConstants = GenesisProtocolConstants
+                { gpcK = 2
+                , gpcProtocolMagic = ProtocolMagic
+                                         (ProtocolMagicId 55550001) RequiresMagic
+                , gpcVssMaxTTL = VssMaxTTL 6
+                , gpcVssMinTTL = VssMinTTL 2
+                }
+            , gsInitializer = GenesisInitializer
+                { giTestBalance = TestnetBalanceOptions
+                    { tboPoors = 12
+                    , tboRichmen = 4
+                    , tboTotalBalance = 600000000000000000
+                    , tboRichmenShare = 0.99
+                    , tboUseHDAddresses = True
+                    }
+                , giFakeAvvmBalance = FakeAvvmOptions
+                    { faoCount = 10
+                    , faoOneBalance = 100000
+                    }
+                , giAvvmBalanceFactor = CoinPortion 100000000000000
+                , giUseHeavyDlg = True
+                , giSeed = 0
+                }
+            }
+        )
+    , ccNtp = NtpConfiguration
+        { ntpcServers =
+            [ "0.pool.ntp.org"
+            , "2.pool.ntp.org"
+            , "3.pool.ntp.org"
+            ]
+        , ntpcResponseTimeout = 30000000
+        , ntpcPollDelay = 1800000000
+        }
+    , ccUpdate = UpdateConfiguration
+        { ccApplicationName = ApplicationName "cardano-sl"
+        , ccLastKnownBlockVersion = BlockVersion 0 0 0
+        , ccApplicationVersion = 0
+        , ccSystemTag = SystemTag "linux64"
+        }
+    , ccSsc = SscConfiguration
+        { ccMpcSendInterval = 10
+        , ccMdNoCommitmentsEpochThreshold = 3
+        , ccNoReportNoSecretsForEpoch1 = False
+        }
+    , ccDlg = DlgConfiguration
+        { ccDlgCacheParam = 500
+        , ccMessageCacheTimeout = 30
+        }
+    , ccTxp = TxpConfiguration
+        { ccMemPoolLimitTx = 200
+        , tcAssetLockedSrcAddrs = S.fromList []
+        }
+    , ccBlock = BlockConfiguration
+        { ccNetworkDiameter = 3
+        , ccRecoveryHeadersMessage = 20
+        , ccStreamWindow = 2048
+        , ccNonCriticalCQBootstrap = 0.95
+        , ccCriticalCQBootstrap = 0.8888
+        , ccNonCriticalCQ = 0.8
+        , ccCriticalCQ = 0.654321
+        , ccCriticalForkThreshold = 2
+        , ccFixedTimeCQ = 10
+        }
+    , ccNode = NodeConfiguration
+        { ccNetworkConnectionTimeout = 15000
+        , ccConversationEstablishTimeout = 30000
+        , ccBlockRetrievalQueueSize = 100
+        , ccPendingTxResubmissionPeriod = 7
+        , ccWalletProductionApi = False
+        , ccWalletTxCreationDisabled = False
+        , ccExplorerExtendedApi = False
+        }
+    , ccWallet = WalletConfiguration { ccThrottle = Nothing }
+    , ccReqNetMagic = RequiresNoMagic
```

Comment:
Had to run this through in my head, but it seems that we're pulling the `"requiresNetworkMagic":"RequiresNoMagic"` field from JSON into `ccReqNetMagic`, and the `ccGenesis.gsProtocolConstants.gpcProtocolMagic.getRequiresNetworkMagic` value is defaulting because the JSON has `"protocolMagic":55550001`. This looks right 👍

Reply:
Thanks for adding this, @intricate.